README.md

```diff
@@ -69,11 +69,15 @@ make -j
 ```
 
 #### Windows
-First, you must [setup vcpkg](https://docs.microsoft.com/en-us/cpp/build/vcpkg?view=vs-2019#installation) and Python 3 to install the dependencies:
+First, you must set up (our version of*) vcpkg and Python 3 to install the dependencies:
```

Comment:
We should probably clarify here the instructions remain the same, just with your fork. Can you also link it?

Reply:
Wouldn't the additional lines afterwards cope for this, since the instructions now state the address to clone from?

My fear is that just stating the directions are the same from the original site will cause them to pull from the MSFT branch and not ours.

Reply:
Ah I did not see those - However I am still concerned about the "completeness" here. Any way we could improve it?

Reply:
I've split the directions up and added a specific link to our fork - pls re-review and let me know if this was more what you were expecting?